tests/functional/base.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+import os
+import shutil
+import subprocess
+import sys
+import time
+import unittest
+from string import Template
+
+import requests
+
+from custodia.client import HTTPUnixAdapter
+from custodia.server.args import parse_args
+from custodia.server.config import parse_config
+
+
+class CustodiaServerBase(unittest.TestCase):
+
+    request_headers = {'REMOTE_USER': 'me'}
+    test_dir = 'tests/functional/tmp'
+
+    @classmethod
+    def setUpClass(cls):
+        super(CustodiaServerBase, cls).setUpClass()
+
+    @classmethod
+    def create_server(cls, configfile=None, configstr=None):
```

Comment:
Yeah, I made this a class method and remove it later, I didn't remember why. I'll fix it.